scripts/data_OEP_registration.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pathlib
+
+import environ_setup
```

Comment:
`environ_setup` is in gitignore and cannot be imported by other users.
Environment vaiables should be set via terminal, IDE or by reading from .env file (therefore [python-dotenv](https://pypi.org/project/python-dotenv/) could be used)